www/content/tutorial.md

```diff
@@ -1517,11 +1533,11 @@ Including the hash solves a number of problems:
 2. Because of 1. there is no need to check the URL on every compilation to see if we have the latest version.
 3. If the domain of the URL expires, a malicious actor can change the package but the hash will not match so the roc cli will reject it.
 
-### [Interface Modules](#interface-modules) {#interface-modules}
+### [Regular Modules](#regular-modules) {#regular-modules}
```

Comment:
@rtfeldman I'm not sure what to call this section now. I thought "Modules" at first, but it feels weird to use the same name as parent section